File: 3731. Find Missing Elements/Solution.py
class Solution:
    def findMissingElements(self, nums: List[int]) -> List[int]:
        if not nums:
            return []
        min_val = min(nums)
        max_val = max(nums)
        nums_set = set(nums)
        missing = []
        for num in range(min_val, max_val + 1):
            if num not in nums_set:
                missing.append(num)
    
        return missing

File: 3731. Find Missing Elements/Solution.js
/**
 * @param {number[]} nums
 * @return {number[]}
 */
var findMissingElements = function(nums) {
    if (nums.length === 0) return [];
    
    const minVal = Math.min(...nums);
    const maxVal = Math.max(...nums);
    const numsSet = new Set(nums);
    
    const missing = [];
    for (let num = minVal; num <= maxVal; num++) {
        if (!numsSet.has(num)) {
            missing.push(num);
        }
    }
    
    return missing;
};
